apps/system/js/browser_context_menu.js

```diff
@@ -207,8 +215,8 @@
             label: _('open-in-new-private-window'),
             callback: this.openUrl.bind(this, uri, true)
           }, {
-            id: 'bookmark-link',
-            label: _('add-link-to-home-screen'),
+            id: 'pin-to-home-screen',
+            label: _('pin-to-home-screen'),
```

Comment:
We may need to only add this when Pin the Web is turned on, until we switch to the new homescreen